egs/wsj/s5/steps/train_mono.sh

```diff
@@ -22,6 +22,7 @@ stage=-4
 power=0.25 # exponent to determine number of gaussians from occurrence counts
```

Comment:
Please revert these changes. We shouldn't change the shared scripts.
Do the same for any other shared script.